components/dashboard/src/settings/Preferences.tsx

```diff
@@ -124,7 +124,7 @@ export default function Preferences() {
                     }
                 </>}
                 {desktopIdeOptions && <>
-                    <h3 className="mt-12">Desktop Editor</h3>
+                    <h3 className="mt-12">Desktop Editor <PillLabel type="warn" className="font-semibold mt-2 py-0.5 px-2 self-center">Beta</PillLabel></h3>
```

Comment:
Note: `self-center` is a `flex` related class (stands for `align-self: center`). It has no effect here, because the parent `<h3>` is not a `flex` container (i.e. it doesn't have the `flex` class, so it's using the default layout, where `align-self` doesn't exist / has no effect).

I think that's why the label is a little low (vertically). To fix it, you could make the `<h3>` a `flex` (also, it's generally preferable to center all the items in the `flex` with a `items-center` class on the parent, as opposed to centering each item separately using `self-center` on the children):

```suggestion
                    <h3 className="mt-12 flex items-center">Desktop Editor <PillLabel type="warn" className="font-semibold mt-2 py-0.5 px-2">Beta</PillLabel></h3>
```

Reply:
Thanks for the insights, @jankeromnes! FWIW, I blindly copied the label component a couple of lines below but definitely worth fixing this alignement.

https://github.com/gitpod-io/gitpod/blob/0e1109f6c3feee202cc5638061e85a22b2433b85/components/dashboard/src/settings/Preferences.tsx#L171

> I think that's why the label is a little low (vertically). 

Thanks for noticing this! Makes me feel a little less crazy. 🧡